driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/FindPublisher.java

```diff
@@ -30,14 +29,7 @@
  * @param <TResult> The type of the result.
  * @since 1.0
  */
-public interface FindPublisher<TResult> extends Publisher<TResult> {
-
-    /**
-     * Helper to return a publisher limited to the first result.
-     *
-     * @return a Publisher which will contain a single item.
-     */
-    Publisher<TResult> first();
```

Comment:
I'm not sure why this was removed.  I see it still in the other publishers.  Maybe it's not needed in the others either?

Reply:
Removed from the others - its provided by the BatchCursorPublisher